SpatialGDK/Source/Private/Interop/GlobalStateManager.cpp

```diff
@@ -188,3 +256,137 @@ bool UGlobalStateManager::IsSingletonEntity(Worker_EntityId EntityId)
 	}
 	return false;
 }
+
+void UGlobalStateManager::ToggleAcceptingPlayers(bool bInAcceptingPlayers)
```

Comment:
Do we ever call this with `false`? If not, just make this `EnableAcceptingPlayers`